internal/export/database/export_test.go

```diff
@@ -102,20 +102,25 @@ func TestLookupSignatureInfos(t *testing.T) {
 			SigningKeyID:      "310",
 		},
 	}
-	for _, si := range want {
+	for _, si := range create {
 		if err := New(testDB).AddSignatureInfo(ctx, si); err != nil {
 			t.Fatalf("failed to add signature info %v: %v", si, err)
 		}
 	}
+	// create a 'reverse' order list.
```

Comment:
You can do this without an extra alloc...

```go
for i, j := 0, len(want)-1; i < j; i, j = i+1, j-1 {
  want[i], want[j] = want[j], want[i]
}
```

Reply:
changed around a bit, still creates an extra alloc... but we're talking about 2 pointers in a unit test